### PR TITLE
OpenVINO dockerfile for ORT

### PR DESCRIPTION
**Pull request by vinitra**
**Description**: Added Intel licenses, package updates, doc updates, and MCR tags in preparation for newest image release for MYRIAD and VAD-M.

**Motivation and Context**
- Including OpenVino images on MCR.

### CODE REVIEW

File: dockerfiles/Dockerfile.openvino [hunk 1]
```diff
@@ -60,6 +59,10 @@ RUN git clone --recursive -b $ONNXRUNTIME_BRANCH $ONNXRUNTIME_REPO /onnxruntime
     /bin/sh onnxruntime/dockerfiles/scripts/install_common_deps.sh && \
```
Comment (jywu-msft) [inline dockerfiles/Dockerfile.openvino]: in line 60, it's missing / in front of onnxruntime/dockerfiles/scripts/install_common_deps.sh
the source was cloned to /onnxruntime in previous line.

Comment (vinitra) [reply dockerfiles/Dockerfile.openvino]: addressed

File: dockerfiles/Dockerfile.openvino [hunk 2]
```diff
@@ -57,9 +56,13 @@ WORKDIR /code
 ENV PATH /opt/miniconda/bin:/code/cmake-3.14.3-Linux-x86_64/bin:$PATH
 
 RUN git clone --recursive -b $ONNXRUNTIME_BRANCH $ONNXRUNTIME_REPO /onnxruntime && \
-    /bin/sh onnxruntime/dockerfiles/scripts/install_common_deps.sh && \
+    /bin/sh /onnxruntime/dockerfiles/scripts/install_common_deps.sh && \
     cd /onnxruntime/cmake/external/onnx && python3 setup.py install && \
     cp /onnxruntime/dockerfiles/LICENSE-IMAGE.txt /code/LICENSE-IMAGE.txt && \
+    cp /onnxruntime/dockerfiles/EULA.txt /code/EULA.txt && \
```
Comment (manashgoswami) [inline dockerfiles/Dockerfile.openvino]: Remove. The EULA.txt is already present in the image from L19 in /opt/intel/openvino_2019.1.144/licensing/EULA.txt ./openvino_2019.1.144/licensing/EULA.txt

File: dockerfiles/Dockerfile.openvino [hunk 3]
```diff
@@ -57,9 +56,13 @@ WORKDIR /code
...
+    cp /onnxruntime/dockerfiles/ISSL.txt /code/ISSL.txt && \
```
Comment (manashgoswami) [inline dockerfiles/Dockerfile.openvino]: /opt/intel/openvino_2019.1.144/licensing/deployment_tools/ISSL.txt

File: dockerfiles/Dockerfile.openvino [hunk 4]
```diff
@@ -57,9 +56,13 @@ WORKDIR /code
...
+    cp /onnxruntime/dockerfiles/redist.txt /code/redist.txt && \
```
Comment (manashgoswami) [inline dockerfiles/Dockerfile.openvino]: /opt/intel/openvino_2019.1.144/licensing/deployment_tools/redist.txt
/opt/intel/openvino_2019.1.144/licensing/openvx/redist.txt
/opt/intel/openvino_2019.1.144/licensing/opencv/redist.txt
/opt/intel/openvino_2019.1.144/licensing/install_dependencies/redist.txt

File: dockerfiles/Dockerfile.openvino (hunk 2 above)
Comment (jywu-msft) [reply dockerfiles/Dockerfile.openvino]: why does it need to be copied if it's already installed and in the image?

File: dockerfiles/EULA.txt [hunk 5]
```diff
@@ -0,0 +1,707 @@
+End User License Agreement for the Intel(R) Software Development Products
```
Comment (jywu-msft) [inline dockerfiles/EULA.txt]: remove this file.

File: dockerfiles/ISSL.txt [hunk 6]
```diff
@@ -0,0 +1,63 @@
+Intel Simplified Software License (Version April 2018)
```
Comment (jywu-msft) [inline dockerfiles/ISSL.txt]: remove this file

File: dockerfiles/redist.txt [hunk 7]
```diff
@@ -0,0 +1,97 @@
+This text file is the file (e.g., redist.txt, etc) specified in the accompanying
```
Comment (jywu-msft) [inline dockerfiles/redist.txt]: remove this file

Comment (manashgoswami) [reply dockerfiles/redist.txt]: @HectorSVC can you please review/approve this PR?

File: dockerfiles/Dockerfile.openvino (hunk 4 above)
Comment (HectorSVC) [reply dockerfiles/Dockerfile.openvino]: my understanding is we should remove these newly added files (EULA.txt, ISSL.txt, redist.txt) from the commit, and remove the cp command lines relate to them as well.

---
In reply to: [327868579](https://github.com/microsoft/onnxruntime/pull/1892#discussion_r327868579) [](ancestors = 327868579)

File: dockerfiles/EULA.txt (hunk 5 above)
Comment (vinitra) [reply dockerfiles/EULA.txt]: removing.

File: dockerfiles/ISSL.txt (hunk 6 above)
Comment (vinitra) [reply dockerfiles/ISSL.txt]: removing

File: dockerfiles/redist.txt (hunk 7 above)
Comment (vinitra) [reply dockerfiles/redist.txt]: removing

**Comment by vinitra** [comment]
Updated with review changes -- removed the license files and the cp commands. @jywu-msft, @HectorSVC  final review?

**Pull request merged**